apps/web/src/app/app/[workspaceSlug]/(dashboard)/settings/appearance/page.tsx

```diff
@@ -12,18 +12,21 @@ export default function AppearancePage() {
 
   return (
     <div className="grid grid-cols-1 gap-4 sm:grid-cols-2 md:grid-cols-4">
+      {/* biome-ignore lint/a11y/useButtonType: */}
       <button onClick={() => setTheme("light")}>
```

Comment:
Interesting - wasn't expecting that you should [always specify](https://www.w3schools.com/jsref/prop_pushbutton_type.asp) the button type.

Reply:
Neither did I. This PR was a nice revisit to the basics.